src/containerapp/HISTORY.rst

```diff
@@ -9,6 +9,7 @@ upcoming
 * 'az containerapp compose create': fixed an issue where the environment's resource group was not resolved from --environment when the input value was a resource id.
 * 'az containerapp replica count', returns the replica count of a container app
 * [Breaking Change] 'az containerapp job create': add default values for container app job properties --replica-completion-count, --replica-retry-limit, --replica-timeout, --parallelism, --min-executions, --max-executions, --polling-interval
+* 'az containerapp env dapr-component init': support initializing Dapr components and dev services for an environment
```

Comment:
please move the history to the `upcoming`.

Reply:
Thanks @Greedygre, let me do it once this PR is reviewed. History file creates a lot of merge conflicts.